build-package.sh

```diff
@@ -712,8 +712,11 @@ termux_step_setup_toolchain() {
 
 		local _LIBDIR=usr/lib
 		if [ $TERMUX_ARCH = x86_64 ]; then _LIBDIR+=64; fi
-		$TERMUX_ELF_CLEANER $_LIBDIR/*.so
-
+		# so we can use aaudio for devices that support it
+		if (( $TERMUX_PKG_API_LEVEL < 26 )); then
```

Comment:
Wonder if `[ "$TERMUX_PKG_API_LEVEL" -lt 26 ]` is better in a way.  At least that errors out properly when the value is not an integer.